### DISCUSSION TITLE
Running proj_ra2ll.csh  has issued warnings

### DISCUSSION THREAD

**Discussion by Congzhangg** [gmtsar/gmtsar#429] [Q&A]
Hi,  @Xiaohua-Eric-Xu ,When I was translating the SBAS results into latitude and longitude coordinates, warnings occurred。Is the result of my SBAS incorrect？

![1662518207348](https://user-images.githubusercontent.com/112800803/188779707-413530a7-dfc5-4fc5-a86b-a9633d0a0b37.png)

**Comment by Xiaohua-Eric-Xu** [comment]
Warnings are OK to ignore as long as your results look fine. GMT wants to warn you that the problem setup may take a while to solve and things will run faster if you choose a loose boundary. Never mind, check on your vel_ll.grd and see whether things look reasonable to you.
